paimon-core/src/test/java/org/apache/paimon/table/RecordLevelExpireTest.java

```diff
@@ -19,17 +19,49 @@
 package org.apache.paimon.table;
 
 import org.apache.paimon.CoreOptions;
+import org.apache.paimon.catalog.Catalog;
+import org.apache.paimon.catalog.CatalogContext;
+import org.apache.paimon.catalog.CatalogFactory;
+import org.apache.paimon.catalog.Identifier;
 import org.apache.paimon.catalog.PrimaryKeyTableTestBase;
 import org.apache.paimon.data.GenericRow;
+import org.apache.paimon.fs.Path;
 import org.apache.paimon.options.Options;
+import org.apache.paimon.schema.Schema;
+import org.apache.paimon.types.DataTypes;
+import org.apache.paimon.utils.TraceableFileIO;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import java.time.Duration;
+import java.util.UUID;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 class RecordLevelExpireTest extends PrimaryKeyTableTestBase {
+    @Override
+    @BeforeEach
+    public void beforeEachBase() throws Exception {
```

Comment:
Why add this method?

Reply:
Column "col1" datatype is `DataTypes.INT()` in PrimaryKeyTableTestBase, but it should be `DataTypes.INT().notNull()` in RecordLevelExpireTest.

Reply:
ok